### PR TITLE
Initial Django install

### LINKED ISSUE #4: Add Django and sandbox setup instructions

### PR DESCRIPTION
**Pull request by fureigh**
This PR closes #4 with a minimal Django install.

Follow-up tasks include: Change `SECRET_KEY` and other sensitive data; load from environment variables instead.

### CODE REVIEW

**Comment by fureigh** [comment]
Optimistically assigning @donjo. :) If you'd prefer to self-assign in the future, just say the word.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+
+This app is designed to run on Python 3.4+. `pyvenv` can manage the dependencies installed with `pip`. With that, you can prepare your development environment by running:
+
+```
+git clone https://github.com/18F/ffd-info-exchange.git
+cd ffd-info-exchange
+pyvenv ffd-env
+source ffd-env/bin/activate
+pip install -r requirements.txt
+createdb ffd-info-exchange
```
Comment (donjo) [inline README.md:34]: maybe a note saying that this is a postgres thing and you have to make sure you have that running? wherever it makes sense to include that. could possibly do a pre-install step that covers all the prep needed to be able to run these commands on the first pass but that could get too environmentally specific

**Review by donjo — COMMENTED** [review]
everything else worked for me. just had to remember I didn't have pg running.

File: README.md (hunk 1 above)
Comment (fureigh) [reply README.md:34]: Good call. I'll see whether there's something preexisting I can leverage/link to.

Comment (fureigh) [reply README.md:34]: Addressed in latest commit; take a look and see what you think. Might be simpler to reduce it to a link to the PostgreSQL docs.

**Comment by fureigh** [comment]
Ready for another look, @donjo.

**Review by donjo — APPROVED** [review]

**Comment by fureigh** [comment]
@donjo Inspired by your Mac specificity, I added another line for non-Mac users. Want to take one more look?

**Comment by donjo** [comment]
@fureigh good call

**Pull request merged**